test/OrchardCore.Tests/OrchardCore.Users/OrchardCore.Users.Core/DefaultUserClaimsPrincipalFactoryTests.cs

```diff
@@ -18,6 +19,7 @@ public class DefaultUserClaimsPrincipalFactoryTests
         [Theory]
         [InlineData(true)]
         [InlineData(false)]
+        [Obsolete]
```

Comment:
This would move to testing the provider factory, not testing the now obsolete factory?